app/controllers/cards_controller.rb

```diff
@@ -37,6 +37,18 @@ def destroy
     respond_with @card
   end
 
+  def check_words
+    if @card.translated_text == card_params[:translated_text].downcase
+      @card.review_date = Time.now + 3.days
+      @card.save
+      flash[:alert] = 'Вы ввели верный перевод. Продолжайте.'
+      redirect_to controller: :home, action: :index
+    else
+      flash[:alert] = 'Вы ввели не верный перевод. Повторите попытку.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.